modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/coverage/io/netcdf/crs/NetCDFProjection.java

```diff
@@ -111,6 +112,29 @@ public String getName() {
     public String getOGCName() {
         return ogcName;
     }
+
+    /**
+     * Subclasses override this if they wish to adjust OGC parameters after they are read from NetCDF. This is the inverse of
+     * {@link #getNetcdfParameters(ParameterValueGroup)}.
+     * 
+     * @param netcdfParameters parameter values read from NetCDF
+     * @return parameter values used for OGC projection
+     */
+    public ParameterValueGroup getOgcParameters(ParameterValueGroup netcdfParameters) {
```

Comment:
The problem with the current implementation is that it assumes the mapping between NetCDF and OGC parameters is a simple renaming. This is not true for the rotated pole projection. These two new methods, whose default implementations do nothing, provide a lightweight mechanism for projection subclasses to change parameter values that is better than adding more ```if``` clauses to ```handleParam```. If we see another case that needs these methods, a more extensive refactoring might be justified.